src/plugins/gui/providers/banxaProvider.ts

```diff
@@ -652,7 +652,7 @@ export const banxaProvider: FiatProviderFactory = {
                                 conversionType: 'sell',
                                 destFiatCurrencyCode: fiatCurrencyCode,
                                 destFiatAmount: priceQuote.fiat_amount,
-                                sourceAmount: CryptoAmount.fromExchangeAmount({
+                                sourceAmount: CryptoAmount({
```

Comment:
Don't you need a `new` here? Also drop the `pluginId`.

Reply:
Global replace fail my bad